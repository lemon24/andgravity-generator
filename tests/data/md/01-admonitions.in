.. warning::

    styles depend on this
