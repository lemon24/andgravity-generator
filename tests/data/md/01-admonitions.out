<section class="admonition warning">
<p class="admonition-title">Warning</p>
<p>styles depend on this</p>
</section>
